### ISSUE TITLE
PHP 8.3 support

### ISSUE THREAD

**Issue by driesvints** [openswoole/ext-openswoole#336]
Heya! We'd like to start verifying Laravel Octane works properly on PHP 8.3 but we can't seem to install openswoole on it. Is a compatible version already available and if not, are there plans to release one soon?

1. What did you do? If possible, provide a simple script for reproducing the error.

Here's a link to a build where openswoole cannot be installed on PHP 8.3: https://github.com/laravel/octane/actions/runs/6614196028/job/18398337797?pr=753

2. What did you expect to see?

A passing build with the openswoole extension installed.

3. What did you see instead?

```
Run php --ri openswoole
Extension 'openswoole' not present.
```

4. What version of OpenSwoole are you using (show your `php --ri openswoole`)?

It can't find a compatible version.

5. What is your machine environment used (show your `uname -a` & `php -v` & `gcc -v`) ?

See "set up job" on the action: https://github.com/laravel/octane/actions/runs/6614196028/job/18398337797?pr=753

**Comment by doubaokun** [comment]
PHP8.3 will be supported soon.

**Comment by doubaokun** [comment]
v22.1.0 is released for PHP 8.1 - PHP 8.3.

**Comment by driesvints** [comment]
Thanks @doubaokun!
